atomata/VClassroom_Proto_VSolar_ApparatusUPackageDevelopment
# New apparatus addressable test

This PR contains the setup for adding the current apparatuses as addressable and a sample scene to load all the assets from azure.
To test: 
build the Container_AddressableTest scene. Click on each button to load the respective apparatus. 
Note: You should get CORS error , which can be solved by adding the Unity Local host address to CORS permission in Azure
